### PR TITLE
AO3-6009 Updated login banner text & links

### PR DESCRIPTION
**Pull request by tlee911**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6009

## Purpose

Updates the text and links from #3913 as requested.
Screen Shot 2020-12-13 at 23 49 18

## Testing Instructions

Feature file for the login banner updated to test this.

For manual testing:
- Create new account or re-enable the first login message
- View the first login banner
- Verify the text and link are displayed correctly
- Verity the link takes user to policy and abuse page

## References

See #3913 for previous work.

### CODE REVIEW

File: config/locales/views/en.yml [hunk 1]
```diff
@@ -99,12 +99,15 @@ en:
...
         dismiss: "Dismiss permanently"
         help_link: "Learn some tips and tricks"
+        help_text: "For more guidance on using the site, %{link_faq}!  If you need technical support, you can %{link_support}. If you experience harassment or have specific questions about the %{link_tos}, you can %{link_abuse}."
         help_title: "First login help"
         hide: "Hide first login help banner"
+        link_abuse: "contact our Policy & Abuse team"
+        link_faq: "check out our FAQ"
+        link_support: "contact us through our Support and Feedback form"
+        link_tos: "Terms of Service"
         welcome_text: "Hi! It looks like you've just logged into the archive for the first time. %{help_link} or dismiss this message permanently."
```
Comment (sarken) [inline config/locales/views/en.yml:111]: Could you capitalize the A in archive here?

**Review by sarken — CHANGES_REQUESTED** [review]
Thank you so much! There was one little issue, but if you don't have time, just let us know and we can take care of it.

**Review by sarken — APPROVED** [review]
Thanks!

**Comment by redsummernight** [comment]
Instead of updating your branch by squashing in a "merge from upstream" pull request (e.g. tlee911/otwarchive#3), I recommend updating it with a normal merge commit (`git pull upstream master`), or just starting a new branch from master entirely. That way your commit history and squashed commit message don't include all the unrelated text/co-authors (061553a).

**Pull request merged**